src/Resources/views/form_layout.html.twig

```diff
@@ -83,7 +85,7 @@
         mask: attr.mask|default('Number'),
         scale: attr.scale|default(2),
         radix: attr.radix|default('.'),
-        thousandsSeparator: attr.thousandsSeparator|default(''),
+        thousandsSeparator: attr.thousandsSeparator is defined ? attr.thousandsSeparator : '\'',
```

Comment:
Nit: Syntactic Sugar

```twig
thousandsSeparator: attr.thousandsSeparator|default('\''),
```

(Unless `thousandsSeparator` could be set as `0`, but I doubt it)

Reply:
That was exactly the tricky part why I had to adjust it. Sorry, should have mentioned it somewhere.

`thousandsSeparator` can pass an empty string which is falsy in Twig and would lead to the default here